Gemfile.lock

```diff
@@ -1,55 +1,55 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.3)
+      actionview (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.3)
+      activesupport (= 6.0.3.3)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activerecord (6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activesupport (6.0.3.2)
+    activemodel (6.0.3.3)
+      activesupport (= 6.0.3.3)
+    activerecord (6.0.3.3)
+      activemodel (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
+    activesupport (6.0.3.3)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2, >= 2.2.2)
     builder (3.2.4)
-    concurrent-ruby (1.1.6)
+    concurrent-ruby (1.1.7)
     crass (1.0.6)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.8.3)
+    i18n (1.8.5)
```

Comment:
### [i18n](https://github.com/ruby-i18n/i18n)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.8.3 → 1.8.5

[_[change-log](https://github.com/svenfuchs/i18n/releases), [source-code](https://github.com/svenfuchs/i18n)_]





<details>
<summary>Commits</summary>

A change of **17** commits. See the full changes on [the compare page](https://github.com/ruby-i18n/i18n/compare/v1.8.3...v1.8.5).

These are the first 10 commits:
- (87b0718) [Fix deprecate_constant INTERPOLATION_PATTERN](https://github.com/ruby-i18n/i18n/commit/87b0718318f50ee006761a4ebe7dd8c923572d67)
- (4e1a134) [Merge pull request #531 from Fatsoma/deprecate_constant_inte…](https://github.com/ruby-i18n/i18n/commit/4e1a134ca4911cd58e9dd97340d78b5f3dc3f52e)
- (76c677a) [Can we call deprecate_constant without checking respond_to?](https://github.com/ruby-i18n/i18n/commit/76c677a783f2d6b77e24e8c0bf842f72859cad53)
- (c2d85c0) [Merge pull request #532 from ruby-i18n/deprecate-constant](https://github.com/ruby-i18n/i18n/commit/c2d85c0bf61c27e1a732c16eebccc1999c6cbaf4)
- (dfd1e29) [Ensure documentation for fallbacks reflects reality](https://github.com/ruby-i18n/i18n/commit/dfd1e29dddeea88a3015b8b1c7ce2b40991a2af8)
- (1cb0987) [Always instantiate I18n::Locale::Fallbacks objects when usin…](https://github.com/ruby-i18n/i18n/commit/1cb0987c8ab2ce4eefcd2310c55ceb62954fccb2)
- (360b8b7) [Merge pull request #535 from ruby-i18n/fallbacks](https://github.com/ruby-i18n/i18n/commit/360b8b74a6abd4f4dead9145912942460e12527d)
- (71a51b3) [Bump to 1.8.4](https://github.com/ruby-i18n/i18n/commit/71a51b3243da7ca65885d2f81d91be15c92402a2)
- (1a4cf20) [Reset load_path to nil in setup](https://github.com/ruby-i18n/i18n/commit/1a4cf2050b758c600255d917868eac4895ac0084)
- (e90c23f) [bugfix custom fallback class](https://github.com/ruby-i18n/i18n/commit/e90c23f5baf527ef5c4ba963f250c4830e3dc439)


</details>